dcrjson/chainsvrcmds.go

```diff
@@ -174,14 +174,16 @@ func NewDecodeRawTransactionCmd(hexTx string) *DecodeRawTransactionCmd {
 
 // DecodeScriptCmd defines the decodescript JSON-RPC command.
 type DecodeScriptCmd struct {
+	Version   uint16
```

Comment:
This is not optional and constitutes a major breaking change that would require a major RPC bump.

Notice the result of `dcrctl` with this:
```
$ ./ctl decodescript "51"
decodescript command: wrong number of params (expected 2, received 1) (code: ErrNumParams)
Usage:
  decodescript version "hexscript"
```